README.md

```diff
@@ -67,7 +67,23 @@ Optional:
 const capabilityAgent = await CapabilityAgent.fromSecret({secret, handle});
 
 // create a keystore and an agent for working with it
-const keystore = KmsClient.createKeystore(...);
+// the baseUrl can be set to a dev API or production API
+const kmsBaseUrl = `${config.server.baseUri}/kms`;
+const keystore = KmsClient.createKeystore({
+  // the url for the keystore is configurable
+  url: `${kmsBaseUrl}/keystores`,
+  config: {
+    // on init the sequence must be 0
+    sequence: 0,
+    controller: capabilityAgent.id,
+    invoker: capabilityAgent.id,
+    // this allows the capabilityAgent to delegate zCaps
+    delegator: capabilityAgent.id
+  },
+  // this can be axios for client side code
+  // or a node https.Agent for back end code
+  httpsAgent
```

Comment:
This comment should be something along these lines:
```
// optional `httpsAgent`, not applicable for front-end, for back-end use cases a configured nodejs `https.Agent` may be to allow the use of self signed certificates using the `rejectUnauthorized: false` flag.
```

Reply:
Just so you know I have had to pass axios as the `httpsAgent` on the front end before. I used most of your comment: https://github.com/digitalbazaar/edv-client/pull/29/commits/4711aeaa59f4ac0c69e121157722395258ab1d37

Reply:
@aljones15 that's alarming.  httpsAgent is a parameter TO axios.  If you can identify where you ran into difficutly with this, please bring it to my attention.

Reply:
Previously in the SecureDataHubClient and the BedrockLedgerClient. It might not be required for LedgerClient, but it was required by the SecureDatahubClient at some point.

Reply:
I just checked and in `edvClient` it appears to use axios everywhere so that problem is probably solved, but I think it might be better for the user to know they can use axios there just in case someone accidentally uses `httpsAgent` to make a call vs passing it to axios.

Reply:
you can't use axios there.